### ISSUE TITLE
Docker - How to pass oauthaccesstoken1 with variable ?

### ISSUE THREAD

**Issue by seb2020** [imapsync/imapsync#465]
Hi,

I have try the docker version of imapsync and I need to use the variable "oauthaccesstoken1". I have set in my docker-compose :

`command: /servimapsync --user1 xxx@mail.com --oauthaccesstoken1 "/tokens/oauth2_tokens_xxx\@mail.com" --host1 outlook.office365.com`

Sadly it seems this parameter and not keep when I try to run the imapsync with the GUI. I can found in the log :
`Command line used, run by /usr/bin/perl:
./imapsync --addheader --automap --host1 outlook.office365.com --justlogin --office1 --password1 MASKED --user1 xxx@mail.com`

Is there a way to use an environment variable when using the webgui ? Or the oauthaccesstoken1 is only supported in the CLI ?

**Comment by gilleslamiral** [comment]
Yes, the CGI mode has a subset of all the parameters, it's a security way, I add a parameter only when I see the need of it and there is no security problem with it.

Look at 
sub get_options_cgi 
sub get_options_cmd

Copy the line you need from get_options_cmd to get_options_cgi and it will be accepted.

I'll add it upstream later.
